frontend/src/views/schedules/Schedules.tsx

```diff
@@ -0,0 +1,59 @@
+import Schedule from '../../components/schedules/Schedule';
+import { ScheduleStatus } from '../../reducers/schedules';
+import { getSchedules } from '../../actions/schedules';
+import { connect } from 'react-redux';
+import { ISchedule } from '../../reducers/schedules';
+import { useEffect } from 'react';
+import Loader from '../../components/commons/Loader';
+import { useNavigate } from 'react-router-dom';
+
+interface ISchedulesProps {
+    getSchedules: Function;
+    isLoading: boolean;
+    schedules: Array<ISchedule>;
+}
+
+const Schedules = ({ getSchedules, isLoading, schedules }: ISchedulesProps) => {
+    useEffect(() => {
+        getSchedules();
+    }, [getSchedules]);
+
+    const navigate = useNavigate();
+
+    const navigateTo = (route: string, state?: object) => {
+        navigate(route, state);
+    };
+
+    return (
+        <section className="flex flex-col h-full min-h-screen w-full bg-gradient-to-b from-bg-gradient-3 to-bg-gradient-4 px-5 lg:px-20 py-10">
+            {isLoading ? (
+                <div className="h-screen flex items-center justify-center">
+                    <Loader size={28} />
+                </div>
+            ) : (
+                <div className="grid grid-cols-1 gap-4 xl:grid-cols-2">
+                    {schedules.map((schedule, index) => (
+                        <Schedule
+                            key={index}
+                            _id={schedule._id}
+                            date={schedule.date}
+                            time={schedule.time}
+                            property={schedule.property}
+                            user={schedule.user}
+                            comments={schedule.comments}
+                            status={schedule.status}
+                            navigateTo={navigateTo}
+                        />
+                    ))}
+                </div>
+            )}
+        </section>
+    );
+};
+
+const mapStateToProps = (state: any) => ({
+    schedules: state.schedules.schedules,
+    isLoading: state.schedules.isLoading,
```

Comment:
importate el ISchedulesState y usalo acá.
```suggestion
const mapStateToProps = (state: { schedules: ISchedulesState }) => ({
    schedules: state.schedules.schedules,
    isLoading: state.schedules.isLoading,
```